src/utilities/utility.ts

```diff
@@ -140,3 +141,24 @@ export function getRandomInt(max: number) {
 export function getCurrentTime(): number {
     return new Date().getTime();
 }
+
+export function resolveGlobPatterns(manifests: string): string[] {
+    if (!manifests) {
+        core.setFailed('No manifests supplied to deploy');
+        return;
+    }
+    let unresolvedManifests = manifests.split('\n');
+    let resolvedManifests: string[] = [];
+    unresolvedManifests.forEach((manifestPath) => {
+        if (glob.hasMagic(manifestPath))
+            resolvedManifests = resolvedManifests.concat(glob.sync(manifestPath));
+        else
+            resolvedManifests.push(manifestPath);
+    });
+
+    if (resolvedManifests == null || resolvedManifests.length == 0) {

```

Comment:
resolvedManifests will never be null. 

You could also just change it to `if (resolvedManifests)`